src/components/shared/Tooltip.css

```diff
@@ -48,17 +48,17 @@
 }
 
 .tooltipHeader {
-  padding-bottom: 10px;
-  margin-bottom: 10px;
+  padding-bottom: 5px;
+  margin-bottom: 5px;
   border-bottom: 1px solid var(--theme-highlight-gray);
 }
 
 .tooltipDetails {
-  margin: 5px 0;
+  display: grid;
+  grid-template-columns: min-content auto;
+  grid-gap: 2px 5px;
```

Comment:
grid all the things !

This makes things better because the first column adjusts itself depending on the label sizes